### DISCUSSION TITLE
how to set nginx?

### DISCUSSION THREAD

**Discussion by forestmojo** [enchant97/web-portal#122] [General]
I want to visit web portal like this way: http://localhost/web , 
so with nginx server configuration, for example:
location /web/ {
                proxy_pass http://localhost:8083/;
        }
nginx is running on port 80. but web portal can't be visited correctly.
Is there any solutions?
thanks a lot for your help.

**Comment by enchant97** [comment]
It looks like you are trying to route with a url prefix which is not supported. If you want multiple services on the same domain; use a sub domain.

**Reply by forestmojo** [reply]
maybe it can work. even with domain name, the port number is still necessary.
but if only IP is supported, not domain name in my case, how to do?

**Reply by enchant97** [reply]
This seems to be an issue not relating specifically to the app. I would suggest to look at some nginx guides, on how to reverse-proxy apps.

Just make sure that any "proxying" you do does not interfere with the URL path.
